ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/InMemoryPackageStore.scala

```diff
@@ -70,6 +73,9 @@ case class InMemoryPackageStore(
     } yield packages
   }
 
+  // FIXME(JM): Move packageEntries into its own interface?
+  def packageEntries(beginOffset: LedgerOffset.Absolute): Source[PackageEntry, NotUsed] = ???
```

Comment:
Forgetmenot

Reply:
Fixed by dropping this and not extending IndexPackageService (which made no sense anyway).